client/src/components/Explorer/Explorer.scss

```diff
@@ -1,8 +1,10 @@
-$c-explorer-bg: #4C4E4D;
+// scss-lint:disable DeclarationOrder
```

Comment:
There was a risk of changing the semantics of the CSS, which I didn't want to take.